### ISSUE TITLE
Renaming ParticleBox

### ISSUE THREAD

**Issue by Fettpet** [ComputationalRadiationPhysics/picongpu#1565]
The particle box in the file _libPMacc/include/particles/memory/boxes/ParticlesBox.hpp_ works  currently with frames and not with particle. It should be renamed.

**Comment by ax3l** [comment]
hi @Fettpet, welcome to PIConGPU! :)

hat's true.

@psychocoderHPC what do you say?

**Comment by psychocoderHPC** [comment]
Yes set's right, the naming of this object is not good. This is from the old times before frames.
We should change it after the release 0.2.0.

**Comment by ax3l** [comment]
not yet included in #1577, so still to do?

**Comment by psychocoderHPC** [comment]
This is still open, I changed it to future.
